### ISSUE TITLE
Filters not working on Bar Charts

### ISSUE THREAD

**Issue by eyounelson** [kenepa/multi-widget#8]
Hello,

We're having issues with combining filters in bar charts within this widget.

The initial load for the bar chart works well. However selecting a filter option throws an component not found error:

<img width="1161" alt="image" src="https://github.com/kenepa/multi-widget/assets/24621192/62e9fa0f-4e12-4043-b613-ca338391bc15">

The bar chart filter works well when in use outside the Dashboard tab widget. So far, we have line widgets in use with this plugin and their filters are working as expected.

**Comment by guerinoj** [comment]
I have the same problem.

**Comment by Jehizkia** [comment]
I'll look into it.

**Comment by guerinoj** [comment]
Thank you. What I have noticed : Filters are working if I have the widget in the same folder than the multi widget one. But it displays twice. If I return false with static function canView(), it doesn't show at all.

**Comment by Jehizkia** [comment]
Could you please provide some code, please? I've tried to reproduce this locally, but unfortunately, I did not manage to do so.

**Comment by guerinoj** [comment]
I tried to reproduce the bug from demo. I can reproduce only one bug : If I return false from canView function, nothing is displayed. Here is the repo to reproduce it : app/Filament/Widgets/LatestOrders.php line 91
https://github.com/guerinoj/demofilament/commit/f016c95f9fb5787b56926bf8d7871fdc3cca3294

**Comment by Jehizkia** [comment]
@guerinoj Which version of multi-widget are you  using?

**Comment by guerinoj** [comment]
This one : "kenepa/multi-widget": "^2.0"
You can have access to repo here: https://github.com/guerinoj/demofilament/tree/multiwidget

**Comment by Jehizkia** [comment]
@guerinoj, I've looked into your issue and understand what's going on. We've changed the way widgets are displayed in #6. The multi-widget checks if it should display your widget based on the `canView()` method in your `LatestOrder.php` widget. So, if `canView()` returns false, it won't display it. If you want to resolve the issue of having a widget displayed twice, you might want to adjust the `discoverWidgets()` function in your `AdminPanelProvider.php` file. This function is responsible for loading the widget an additional time.

This is my fault because I forgot to update the documentation and didn't assign the new update to the proper tag. Sorry for the inconvenience.
